app/views/patients/_patient_dashboard.html.erb

```diff
@@ -22,7 +22,7 @@
 
   <div class="col-sm-4">
     <div class="row">
-      <%= f.text_field :appointment_date, label: 'Appointment date', autocomplete: 'off', format: '%Y-%m-%d', placeholder: 'YYYY-MM-DD', help: "Approx gestation at appt: #{patient.last_menstrual_period_at_appt}" %>
+      <%= f.text_field :appointment_date, value: patient.appointment_date_display, label: 'Appointment date', autocomplete: 'off', format: '%m-%d-%Y', placeholder: 'MM-DD-YYYY', help: "Approx gestation at appt: #{patient.last_menstrual_period_at_appt}" %>
```

Comment:
We can change `f.text_field` to `f.date_field`, have the placeholder use slashes instead of dashes, and ditch value and format I think. I'm good with getting us 90% of the way there and can live without hyphens.